test/built-ins/Temporal/ZonedDateTime/prototype/until/roundingincrement-non-integer.js

```diff
@@ -20,5 +20,3 @@ const earlier = new Temporal.ZonedDateTime(1_000_000_000_000_000_000n, "UTC");
 const later = new Temporal.ZonedDateTime(1_000_000_000_000_000_005n, "UTC");
 const result = earlier.until(later, { roundingIncrement: 2.5, roundingMode: "trunc" });
 TemporalHelpers.assertDuration(result, 0, 0, 0, 0, 0, 0, 0, 0, 0, 4, "roundingIncrement 2.5 truncates to 2");
-const result2 = earlier.until(later, { smallestUnit: "days", roundingIncrement: 1e9 + 0.5, roundingMode: "expand" });
-TemporalHelpers.assertDuration(result2, 0, 0, 0, 1e9, 0, 0, 0, 0, 0, 0, "roundingIncrement 1e9 + 0.5 truncates to 1e9");
```

Comment:
Do we want to test for a RangeError for ridiculous roundingIncrements in ZonedDateTime.p.since/until? Maybe you have this somewhere else, apologies. Here's the test code fullcalendar used:
https://github.com/fullcalendar/test262/commit/887e0b5531d0b3375053a634f6119c5dc7743967

Reply:
Yeah, that's a good point. I removed this from here because with this method it is no longer possible to test that 1e9 + 0.5 gets truncated (the RangeError that would be thrown if it isn't truncated is indistinguishable from the RangeError that is thrown when the addition fails, from a spec point of view). But the test you mentioned should be added elsewhere. I've pushed a new commit to this PR adding one.